spec.bs

```diff
@@ -597,41 +614,39 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     [=auction config/per buyer experiment group ids=].
   1. If |perBuyerExperimentGroupIds| is not null and |perBuyerExperimentGroupIds|[|buyer|] exists:
     1. Set |buyerExperimentGroupId| to |perBuyerExperimentGroupIds|[|buyer|].
-  1. Let |perBuyerSignals| be |auctionConfig|'s [=auction config/per buyer signals=][|buyer|] if that
-    member [=map/exists=], and null otherwise.
-  1. If |auctionConfig|'s [=auction config/per buyer signals=] is not null and [=auction config/per buyer signals=][|buyer|] [=map/exists=]:
-    1. Let |perBuyerSignals| be |auctionConfig|'s [=auction config/per buyer signals=][|buyer|].
-  1. Otherwise:
-    1. Let |perBuyerSignals| be null.
+  1. Let |perBuyerSignals| be null.
+  1. If |auctionConfig|'s [=auction config/per buyer signals=] is not null and
+    [=auction config/per buyer signals=][|buyer|] [=map/exists=]:
+    1. Set |perBuyerSignals| to |auctionConfig|'s [=auction config/per buyer signals=][|buyer|].
   1. [=map/For each=] |signalsUrl| -> |perSignalsUrlGenerator| of |perBuyerGenerator|:
     1. Let |keys| be a new [=ordered set=].
     1. Let |igNames| be a new [=ordered set=].
     1. [=map/For each=] joiningOrigin -> |groups| of |perSignalsUrlGenerator|:
       1. [=list/For each=] |ig| of |groups|:
         1. [=set/Append=] |ig|'s [=interest group/trusted bidding signals keys=] to |keys|.
         1. [=set/Append=] |ig|'s [=interest group/name=] to |igNames|.
-    1. Let |fullSignalsUrl| be the result of [=building trusted bidding signals url=] with
+    1. Let |biddingSignalsUrl| be the result of [=building trusted bidding signals url=] with
       |signalsUrl|, |keys|, |igNames|, |buyerExperimentGroupId|.
-    1. Let |trustedBiddingSignals| be the result of [=fetching resource=] with |fullSignalsUrl|,
-      "application/json", and null.
+    1. Let « |trustedBiddingSignals|, |dataVersion| » be the result of [=fetching trusted signals=]

```

Comment:
The trusted bidding signals fetch also returns the priority vector in the response -- not sure how we want to integrate that here?

Reply:
This led me down a rabbit-hole...
1. I reworked trustedBiddingSignals as it didn't contain the right values.
2. I added code to handle formatVersionString.
Now back to your question, we aren't handling priority vectors yet, so for now they're ignored in trusted bidding signals fetches.  Supporting priority vector reprioritization in the spec is on our spec TODO list.